app/views/submissions/steps/trial/_step04_before.html.haml

```diff
@@ -91,7 +93,6 @@
         Please use the values from the <b>Variety</b> column when referring to existing plant varieties
         in your CSV file in order to avoid duplications due to differences in spelling.
 
-  %br
   %h5 Trait scores ordering #{'and technical replicates' if rep_traits.present?}
   %p
     The first line will be treated as a header. Specific values in columns
```

Comment:
There is something wrong with column ordering in this info box. In my case it says the trait scoring starts in column 5 while the template has its first trait scoring column at index 6 (starting with index 1, as a user would do). Please check again if we are counting these columns correctly in this info box.

Reply:
You are right, the numbers were not updated when we added "Year produced" column.